pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -3959,6 +3959,7 @@ broken-packages:
   - Dao
   - dao
   - dapi
+  - darcs
```

Comment:
As long as `darcs` is working now with the changes in this PR, it doesn't need to be in this list.

Reply:
ok, it's removed now